### ISSUE TITLE
the program does not work as the paper described?

### ISSUE THREAD

**Issue by yanshengli** [gssci/neural-graph-network-sentiment-analysis#1]
hello,
    i run your program ,bug i get the result below:

======== EPOCH 10 ========
Step: 13500 | Last Batch Accuracy: 0.56626505 | Epoch Avg Accuracy: 0.5020844 | Batch Test Accuracy: 0.508 | Train Loss: 439.75624799728394
Step: 13600 | Last Batch Accuracy: 0.516129 | Epoch Avg Accuracy: 0.49786854 | Batch Test Accuracy: 0.49800003 | Train Loss: 1044.2153811454773
Step: 13700 | Last Batch Accuracy: 0.5119048 | Epoch Avg Accuracy: 0.49930593 | Batch Test Accuracy: 0.5093334 | Train Loss: 1664.813934803009
Step: 13800 | Last Batch Accuracy: 0.46341464 | Epoch Avg Accuracy: 0.5007814 | Batch Test Accuracy: 0.504 | Train Loss: 2283.0844588279724
Step: 13900 | Last Batch Accuracy: 0.4615385 | Epoch Avg Accuracy: 0.5009949 | Batch Test Accuracy: 0.5056 | Train Loss: 2893.9820761680603
Step: 14000 | Last Batch Accuracy: 0.5 | Epoch Avg Accuracy: 0.5015032 | Batch Test Accuracy: 0.49266663 | Train Loss: 3513.5789852142334
Step: 14100 | Last Batch Accuracy: 0.494382 | Epoch Avg Accuracy: 0.50256604 | Batch Test Accuracy: 0.49428567 | Train Loss: 4138.261017799377
Step: 14200 | Last Batch Accuracy: 0.48888892 | Epoch Avg Accuracy: 0.5022914 | Batch Test Accuracy: 0.488 | Train Loss: 4758.759525775909
Step: 14300 | Last Batch Accuracy: 0.46052632 | Epoch Avg Accuracy: 0.50288683 | Batch Test Accuracy: 0.48444447 | Train Loss: 5360.021762371063
Step: 14400 | Last Batch Accuracy: 0.48235297 | Epoch Avg Accuracy: 0.50364584 | Batch Test Accuracy: 0.49120003 | Train Loss: 5960.287702560425
Step: 14500 | Last Batch Accuracy: 0.51282054 | Epoch Avg Accuracy: 0.5034041 | Batch Test Accuracy: 0.4938182 | Train Loss: 6525.930757045746
Step: 14600 | Last Batch Accuracy: 0.4044944 | Epoch Avg Accuracy: 0.50334364 | Batch Test Accuracy: 0.49433336 | Train Loss: 7085.406883716583
Step: 14700 | Last Batch Accuracy: 0.5308642 | Epoch Avg Accuracy: 0.5026719 | Batch Test Accuracy: 0.4963077 | Train Loss: 7596.605616092682
Step: 14800 | Last Batch Accuracy: 0.4555556 | Epoch Avg Accuracy: 0.5028481 | Batch Test Accuracy: 0.49457145 | Train Loss: 8116.533761024475
Step: 14900 | Last Batch Accuracy: 0.5529412 | Epoch Avg Accuracy: 0.5031531 | Batch Test Accuracy: 0.4938667 | Train Loss: 8676.518471717834
 
 i want to know  your results in your computer
i am best regard for your reply

**Comment by gssci** [comment]
Hello,
what you are running is the training script, starting from epoch 10 by loading a previous model. 

If you would like to try out the program, run "demo.py"
